examples/catalog/src/content/card/CardExample.scss

```diff
@@ -0,0 +1,4 @@
+:local .container {
```

Comment:
what is this :local?

Reply:
This keyword generate random className!

Reply:
This is a sass function?

Reply:
👍

Reply:
Gotcha - I've used that before but not in production. Great feature!